### DISCUSSION TITLE
Can't use my github to sign in

### DISCUSSION THREAD

**Discussion by Smartosky** [community/community#36715] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, please I opened an account on Aptoslabs with mt github account linked, I can't access the Aptoslab account now using the github account. This is the error message I'm getting - Could not authenticate you from GitHub because "Your application has been suspended. please visit https://support.github.com.".

**Comment by jooy2** [comment, marked as answer]
I am also experiencing the same problem, so it seems that the login is failing due to a temporary error in the Aptoslabs service.

Looks like I'll have to wait for the login issue to be resolved, or send an inquiry to Aptoslabs' customer support.

If it was helpful, please mark it as an answer. Thank you.

**Comment by ASSOC-exe** [comment]
I have an identical problem with Aptos Labs authorization. Tested from different devices and browsers.

**Comment by blueOMG** [comment]
I met this problem again, how can i solve it?

**Comment by HabibMaula** [comment]
Yeah same with you, I can't log in via GitHub

**Comment by googra** [comment]
me either. the same problem.

**Comment by llliiiuuuzzz** [comment]
Why the way of logging by github is gone?

**Comment by A-L-E-XX** [comment]
Authenticating failed due to the following error: Your account requires additional verification. Please check your email and follow the instructions to verify your account.
